src/workspace/views/BookmarksOverview.tsx

```diff
@@ -335,6 +386,11 @@ const BookmarksOverview: FC<BookmarksOverviewProps & UserProps & RouteComponentP
 	const renderBookmarks = () => (
 		<>
 			{bookmarks && bookmarks.length ? renderTable() : renderEmptyFallback()}
+			<BookmarkEmbedModal
+				isOpen={!!isEmbedCodeModalOpen}
+				embedCode={isEmbedCodeModalOpen}
+				onClose={() => setIsEmbedCodeModalOpen(undefined)}
```

Comment:
PERSONAL PREFERENCE:
for intentional clearing of variables i prefer null
things that are accidentally undefined i prefer undefined
```suggestion
				onClose={() => setIsEmbedCodeModalOpen(null)}
```